README.md

```diff
@@ -43,17 +43,25 @@ The plugin can be configured in the [**semantic-release** configuration file](ht
   "plugins": [
     "@semantic-release/commit-analyzer",
     "@semantic-release/release-notes-generator",
-    "semantic-release-hackage"
+    [
+      "semantic-release-hackage",
+      {
+        "cabalFile": "aeson",
+        "packageName": "aeson",
```

Comment:
Should we use a more generic name? This could be confusing, people might think that's the name they need to use, wdyt?